pulseapi/entries/test_views.py

```diff
@@ -554,6 +559,29 @@ def test_moderation_toggle_by_staff(self):
         entry = Entry.objects.get(id=entry_id)
         self.assertEqual(entry.moderation_state, state)
 
+    def test_entry_serializer(self):
+        """
+        Make sure that the entry serializer contains all the custom data needed.
+        Useful test to make sure our custom fields are tested and not
+        removed during API changes
+        """
+
+        entries = self.entries
+        serialized_entries = EntrySerializer(entries, many=True).data
+
+        for i in range(len(serialized_entries)):
+            entry = entries[i]
```

Comment:
Clearly this works, but is there a documented reason why the ordering of `serialized_entries` and `entries` match? (I don't see us forcing an ordering, so a future model change might break this)

Reply:
It matches because we use `entries` to populate `serialized_entries` (passed as the first param to the serializer). So whatever order `entries` is in, `serialized_entries` will follow (regardless of model ordering changes).